class-two-factor-core.php

```diff
@@ -593,6 +614,31 @@ public static function backup_2fa() {
 		exit;
 	}
 
+	/**
+	 * Displays a message informing the user that their account has had failed login attempts.
+	 *
+	 * @param WP_User $user WP_User object of the logged-in user.
+	 */
```

Comment:
```suggestion
	 *
	 * @codeCoverageIgnore
	 */
```

Reply:
I personally don't like using codeCoverageIgnore, I consider untestable functions should simply result in the coverage numbers being lower.

That being said, I can probably add a unit test for this..

Reply:
Either way 👍🏻

Reply:
Unit test added.